Adding a note for content staging preview (#1363)
magento/merchdocs: src/cms/content-staging-preview.md

   - Paste the link into the body of an email message.

     {:.bs-callout-info}

>      Only Admin users are allowed to see the preview, you are sharing. It is required to create admin account for user before the sharing, if it does not exist.Otherwise, user won’t be able to see a preview.

Reviewer: ```suggestion
     An Admin user account is required to see a shared preview. If you have access to [create a new Admin user account]({% link system/permissions-users-all.md %}), you must create the account for a new user before the sharing.
```
